core/src/main/java/jenkins/security/ClassFilterImpl.java

```diff
@@ -305,11 +314,22 @@ public boolean isBlacklisted(String name) {
                 LOGGER.log(Level.WARNING, "would normally reject {0} according to standard blacklist; see https://jenkins.io/redirect/class-filter/", name);
```

Comment:
It seems inconsistent to not log the stack trace when the blacklist is suppressed since you do log it if the whitelist is suppressed.

Reply:
Yes, I forgot it